modules/negotiate/lib/Auth/Source/Negotiate.php

```diff
@@ -136,6 +138,12 @@ public function authenticate(&$state)
                     SimpleSAML\Logger::debug('Negotiate - authenticate(): No "Negotiate" found. Skipping.');
                 }
             }
+            
+            if ($this->rcache === true) {
+                putenv("KRB5RCACHETYPE=dfl");
+            } elseif ($this->rcache === false) {
```

Comment:
Would it make more sense to make this a simple `else` without any condition? What's the default in case the config option is not is not set?